src/components/NoData.tsx

```diff
@@ -4,7 +4,7 @@ import '@styles/NoData.styles.scss'
 export const NoData = () => {
   return (
     <div className='no-data-container'>
-      <img className='cat-img' src={CryingCat} />
+      <img className='cat-img' alt='Loading' src={CryingCat} />
```

Comment:
I believe this should say "no data" not loading

Reply:
Resolved